.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "C++ Launch",
```

Comment:
Name should be more descriptive than the generic "C++ Launch" such as  "debug detectimg".